package.json

```diff
@@ -20,9 +20,14 @@
   "homepage": "https://github.com/knight-18/Medical-Records#readme",
   "dependencies": {
     "bcryptjs": "^2.4.3",
+    "check-password-strength": "^1.0.15",
+    "connect-flash": "^0.1.1",
+    "cookie-parser": "^1.4.5",
     "dotenv": "^8.2.0",
     "ejs": "^3.1.5",
     "express": "^4.17.1",
+    "express-messages": "^1.0.1",
```

Comment:
Why are you using `express-messages`?

Reply:
express-messages was used for flash messages. It is a utility

Reply:
That can be implemented without this also. This package has very less downloads.

Reply:
I see, bhaiya. What would you recommend for flash messages then? Connect - flash has a lot of downloads I guess. 
Express-messages is built on top of connect-flash though.

Reply:
Better go with 'connect-flash'. 'express-messages' just has 4 versions released in the past 10 years.

Reply:
I have gotten rid of express-messages and have changed the code so that it only uses connect-flash